lib/phlex/static_component.rb

```diff
@@ -0,0 +1,24 @@
+module Phlex
+  class StaticComponent < Component
+    def call
+      return super if is_a? Compiler
+      if respond_to? :compiled_template
```

Comment:
We need a faster way to detect if a compiled template already exists. I think `respond_to?` is quite slow because it needs to look through the ancestry.

Reply:
Maybe set instance variable once `compiled_template` is called ?